packages/larva-tokens/build/variety.custom-properties.css

```diff
@@ -0,0 +1,33 @@
+
+:root {
+  --color-white: rgb(255, 255, 255);
+  --color-grey-light: rgb(230, 230, 230);
+  --color-grey: rgb(140, 140, 140);
+  --color-grey-dark: rgb(106, 106, 106);
+  --color-black: rgb(0, 0, 0);
+  --font-family-primary-fancy: 'Para Supreme Regular', serif;
+  /* Fallback for primary-fancy */
+  --font-family-primary: Georgia, serif;
+  --font-family-secondary-fancy: 'IBM Plex Sans', sans-serif;
+  /* Fallback for secondary-fancy */
+  --font-family-secondary: Helvetica, serif;
+  --font-family-body: Georgia, serif;
+  --font-family-basic: Helvetica, sans-serif;
+  --font-family-accent: Helvetica, sans-serif;
+  --background-color-body: rgb(255, 255, 255);
+  --background-color-white: rgb(255, 255, 255);
+  --background-color-grey-lightest: rgb(247, 247, 247);
+  --background-color-grey-light: rgb(204, 204, 204);
+  --background-color-grey: rgb(140, 140, 140);
+  --background-color-grey-dark: rgb(106, 106, 106);
+  --background-color-grey-darkest: rgb(50, 50, 50);
+  --background-color-black: rgb(0, 0, 0);
+  --color-brand-secondary: rgb(65, 126, 244);
+  --font-family-basic-fancy: 'IBM Plex Mono', monospace;
+  --color-brand-primary: rgb(238, 232, 9);
+  --font-family-body-fancy: 'IBM Plex Serif', serif;
+  --border-color-brand-primary: rgb(65, 126, 244);
+  --background-color-brand-primary: rgb(238, 232, 9);
+  --background-color-brand-accent-a-100-b: rgb(26, 40, 47);
```

Comment:
Where is this coming from? What does `a-100-b` represent?

For colors, I'm also guessing there will be several for VY that are not in the existing tokens. Added some guidelines for that here: https://github.com/penske-media-corp/pmc-larva/wiki/CSS-Class-Generators#basic-utility-generator